packages/transaction-controller/src/utils/history.ts

```diff
@@ -7,6 +7,22 @@ import type {
   TransactionMeta,
 } from '../types';
 
+/**
+ * The maximum allowed length of the `transaction.history` property.
+ */
+export const MAX_HISTORY_LENGTH = 100;
+
+/**
+ * A list of the paths used in the display of a transaction activity log.
+ */
+export const ACTIVITY_LOG_HISTORY_PATHS = [
```

Comment:
Is there an argument to decouple, even conceptually, from what the clients chose to do with the history data?

Could this be `UNCOMPRESSABLE_HISTORY_PATHS` for example?

Logically, we could also accept these as a constructor argument but that feels excessive if we ultimately plan to remove the history in the long term.

Reply:
>Logically, we could also accept these as a constructor argument but that feels excessive if we ultimately plan to remove the history in the long term.

Yeah, this was my thought exactly.

We could certainly rename it. But I was trying to capture _why_ we're not compressing it. Really these entries _can_ be compressed just as well as the others. We'd lose some information (the timestamp), but that's true of the others as well. So calling them "uncompressable" doesn't seem exactly right.

We could call it something more generic like `DISPLAYED_HISTORY_PATHS` - paths used for display by clients. Any other ideas?

Reply:
I have renamed it in a1cd8d91 to be `DISPLAYED_TRANSACTION_HISTORY_PATHS`, which is at least more consistent with other internal variables, and a bit less specific to how it's used (the description also now says that these paths "may" be displayed, not that it's certain they are).